src/lib/component/logger.hpp

```diff
@@ -55,6 +55,10 @@ namespace tags {
     template <typename T>
     struct plot_type {};
 
+    //! @brief Declaration tag associating to an output stream type
+    template <typename O>
```

Comment:
By convention the typename parameter should be called `T` (it is so for every declaration tag)